add obsidian-etherpad-plugin to community-plugins.json

egradman opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:

https://github.com/egradman/obsidian-etherpad-lite

## Release Checklist
- [ ] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [ ] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [ ] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

lishid commented:
https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L4
Use the YAML functions from obsidian rather than importing your copy to minimize your bundle size.

https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L10
Use `let` instead of `var`.

https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L88
You should use `view.file` instead of getting a new copy from workspace.

https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L93
You can get the text content via `editor.getValue()` instead of having to read from disk.

https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L116
Should await on this.

https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L124
Same as above.

https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L131
Don't use this API. `openExternal` is quite dangerous, plus the Electron API doesn't exist on mobile. You can simply use `window.open(url)` instead.

https://github.com/egradman/obsidian-etherpad-lite/blob/f4204811bd38d7ffefbbd42ba9f048fbf09dffcc/main.ts#L216
Looks unnecessary?

egradman commented:
Thank you for the thorough code review!

I've addressed all issues you flagged, and re-tested everything.  I've created a new release (1.0.5)

Thanks.
